acceptance/selftest/record_cloud/workspace-file-io/test.toml

```diff
@@ -1,9 +1,4 @@
-# hello, world\n base64 encoded in unix
+# hello, world\n base64 encoded
 [[Repls]]
 Old = "aGVsbG8sIHdvcmxkCg=="
 New = "[HELLO-WORLD]"
```

Comment:
Why not drop this replacement as well? It's not dynamic, so a comment in the script will do.

Reply:
It's slightly more robust. A comment would be easy to miss if the actual uploaded value changes. People might miss a base64 encoding change if, for example, someone decides to refactor the test.

It's also what we do for base64 encoded strings in other places.

I don't mind either way though, happy to go with a comment too.

Reply:
For me, the less replacements there are, the easier it is to read the test output.